samigo/samigo-app/src/webapp/jsf/delivery/discrepancyInData.jsp

```diff
@@ -58,11 +58,12 @@
 
  <h:form id="discrepancyInData">
  <p class="act">
-       <h:commandButton value="#{deliveryMessages.button_return}" type="submit"
-         styleClass="active" action="select" rendered="#{delivery.actionString=='takeAssessment'}">
-          <f:actionListener
-            type="org.sakaiproject.tool.assessment.ui.listener.select.SelectActionListener" />
+       <h:commandButton value="#{deliveryMessages.button_return}" type="submit" styleClass="active" action="select" rendered="#{delivery.actionString=='takeAssessment'}">
+          <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.select.SelectActionListener" />
        </h:commandButton>
+       <h:outputLink rendered="#{delivery.actionString == 'takeAssessmentViaUrl'}" value="#{delivery.getPublishedURL()}">
```

Comment:
should this be delivery.publishedURL instead of delivery.getPublishedURL() ?

Reply:
No, publishedURL isn't a member of the bean, it's a newly introduced method. Withouth the `()` it tries to access a member variable of that name via standard getter. At least this is my understanding of JSF. It does work as is though.

Reply:
yeah that's the correct understanding AFAIK!